lighthouse-cli/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "scripts": {
```

Comment:
does it make sense to add a one-off compile to the parent module's postinstall and prepublish?